### PR TITLE
Update mod to Minecraft 1.19.4

### PR DESCRIPTION
**Pull request by haykam821**
Note that this pull request contains breaking changes to match the supplier changes in DataFixerUpper itself.

### CODE REVIEW

File: src/main/java/xyz/nucleoid/codecs/MoreCodecs.java [hunk 1]
```diff
@@ -187,7 +187,7 @@ public static <T> Codec<T> validate(Codec<T> codec, Function<T, DataResult<T>> v
         return codec.flatXmap(validate, validate);
     }
 
-    public static <T> Codec<T> validate(Codec<T> codec, Predicate<T> validate, String error) {
+    public static <T> Codec<T> validate(Codec<T> codec, Predicate<T> validate, Supplier<String> error) {
```
Comment (Gegy) [inline src/main/java/xyz/nucleoid/codecs/MoreCodecs.java:162]: I think we can deprecate this now, those exist in `Codecs` too 🙂

**Review by Gegy — COMMENTED** [review]
Do we want to drop deprecated codecs with this breaking change?

**Comment by haykam821** [comment]
I've removed the previously-deprecated codecs in #5.

File: src/main/java/xyz/nucleoid/codecs/MoreCodecs.java (hunk 1 above)
Comment (haykam821) [reply src/main/java/xyz/nucleoid/codecs/MoreCodecs.java:162]: Fixed in f7b2682f7a76e30ce1b4b61d4bf91f1a1aa0bee7

**Review by Gegy — APPROVED** [review]

**Pull request merged**